lib/ansible/modules/windows/win_lineinfile.ps1

```diff
@@ -95,10 +95,10 @@ function WriteLines($outlines, $path, $linesep, $encodingobj, $validate) {
 		}
 
 	}
-	
+
 	# Commit changes to the path
 	$cleanpath = $path.Replace("/", "\");
-	Copy-Item $temppath $cleanpath -force;	
+	Copy-Item $temppath $cleanpath -force;
```

Comment:
consider adding try .. .catch block to capture any error messages and fail gracefully.